public/docs/_examples/dependency-injection/dart/lib/logger_service.dart

```diff
@@ -3,10 +3,12 @@ import 'package:angular2/core.dart';
 
 @Injectable()
 class Logger {
-  List<String> logs = [];
+  List<String> _logs = [];
+
```

Comment:
dartfmt doesn't seem to care. It leaves the line there if present, and doesn't add it if it is absent. I'll all for compactness, so I removed that extra line.